HA.md

```diff
@@ -0,0 +1,21 @@
+# High Availability (HA)
+## Introduction
+When we create kubernetes cluster using kops we are able to create a multiple K8S masters and by doing so we can assure that in a case of 
+a master instance failure our K8S functionality won't damaged. 
+In scenario of a single master instance that fails our minions/pods will keep running but there will be no new pods schedualing, kubectl
+won't work (since the api-server will be down).
+
+
+## Kops HA
+We can create HA cluster using kops only in the first creation of the cluster by using the "--master-zones" flag
+example: https://github.com/kubernetes/kops/blob/master/docs/advanced_create.md
+K8S relay on a key value DB named "etcd", which is using the the Qurom concept that the cluster need at least 51% of nodes to be
```

Comment:
Here is some more data:

Etcd uses the Raft consensus algorithm to maintain consistency. The algorithm requires that a quorum of nodes be in-sync to make a decision. If you have an n-node cluster, you’ll need n / 2 + 1 nodes for a quorum.

If you want to rewrite that so we are not plagiarizing :)